docs/outputs.rst

```diff
@@ -94,6 +94,68 @@ betas_hik_OC_T1c.nii      T1-GS corrected high-kappa components
 
+Component tables
+----------------
+TEDPCA and TEDICA use tab-delimited tables to track relevant metrics, component
+classifications, and rationales behind classifications.
+TEDPCA rationale codes start with a "P", while TEDICA codes start with an "I".
+
+===============    =============================================================
+Classification     Description
+===============    =============================================================
+accepted           BOLD-like components retained in denoised and high-Kappa data
+rejected           Non-BOLD components removed from denoised and high-Kappa data
+retained           Low-variance components retained in denoised, but not
+                   high-Kappa, data
+===============    =============================================================
+
+TEDPCA codes
+````````````
+
+=====  ===============  ===============================  =======================
+Code   Classification   Description                      Algorithm(s)
+=====  ===============  ===============================  =======================
+P001   rejected         Low Rho, Kappa, and variance     Kundu decision tree
+                        explained
+P002   rejected         Low variance explained           Kundu decision tree
+P003   rejected         Kappa equals fmax                Kundu decision tree
+P004   rejected         Rho equals fmax                  Kundu decision tree
+P101   rejected         Cumulative variance explained    Kundu decision tree
+                        above 95%                        (stabilized version)
+P102   rejected         Kappa below fmin                 Kundu decision tree
+                                                         (stabilized version)
+P103   rejected         Rho below fmin                   Kundu decision tree
+                                                         (stabilized version)
+=====  ===============  ===============================  =======================
+
+TEDICA codes
+````````````
+=====  ===============  ===============================  =======================
+Code   Classification   Description                      Algorithm(s)
+=====  ===============  ===============================  =======================
+I001   rejected         Manual exclusion                 All
+I002   rejected         Rho greater than Kappa or        Kundu v2.5, Kundu v3.2
+                        more significant voxels
+                        in S0 model than R2 model
+I003   rejected         S0 Dice is higher than R2 Dice   Kundu v2.5, Kundu v3.2
+                        and high variance explained
+I004   rejected         Noise F-value is higher than     Kundu v2.5, Kundu v3.2
+                        signal F-value and
+                        high variance explained
+I005   retained         No good components found         Kundu v2.5
+I006   rejected         Mid-Kappa component              Kundu v2.5, Kundu v3.2
```

Comment:
I don't think we should be putting in the ICA selection in the docs, since we're not supporting v3.2 right now. Should we put these in a gist for now, so we can come back to them ?

Reply:
That makes sense. You made the v3.2 gist, right? Would you mind making this one too, so that they're easier to find together?

Reply:
Created ! https://gist.github.com/emdupre/f65999c4952f1e0b499eee9e0e61ef0e